server/server/src/router_service.rs

```diff
@@ -97,6 +98,27 @@ pub fn core_routes(
         .or(core_req!(GetPublicKeyRequest, get_public_key, server_state))
         .or(core_req!(GetUsageRequest, get_usage, server_state))
         .or(core_req!(GetUpdatesRequest, get_updates, server_state))
+        .or(core_req!(
+            RegisterCreditCardRequest,
+            register_credit_card,
+            server_state
+        ))
+        .or(core_req!(
+            RemoveCreditCardRequest,
+            remove_credit_card,
+            server_state
+        ))
+        .or(core_req!(
+            GetRegisteredCreditCardsRequest,
```

Comment:
We will likely want this to be more generally usable across the various forms of billing, but we can tackle that once we have other forms of billing.